### PR TITLE
Fixing docker-compose webpack compatibility

### LINKED ISSUE #199: Docker Compose - unexpected token at 'Invalid Host header'
Hi!
I've download the repository and launched as it's written for docker:

> # Checkout the project
> $ git clone git://github.com/Codeminer42/cm42-central.git
> $ cd cm42-central
> 
> # copy and edit the configuration
> $ cp .env.sample .env
> $ cp config/database.yml.example config/database.yml
> 
> # Prepare container
> $ docker-compose build
> $ docker-compose run --rm web npm install
> $ docker-compose run --rm web bundle exec rake db:create
> $ docker-compose run --rm web bundle exec rake db:migrate
> $ docker-compose run --rm web bundle exec rake db:seed
> 
> # Up container
> $ docker-compose up
> 

I cannot start the app:

error

The log from the starting up the app is in the attachment - after:
> $ git clone git://github.com/Codeminer42/cm42-central.git
> $ cd cm42-central
> 
> # copy and edit the configuration
> $ cp .env.sample .env
> $ cp config/database.yml.example config/database.yml

There is the log from terminal (running on Mac OS):
[docker-output.txt](https://github.com/Codeminer42/cm42-central/files/1072937/docker-output.txt)

Thank you in advance.

### PR DESCRIPTION
**Pull request by leonardofreitass**
Adding `--public webpack`to `docker-compose.yml` to fix #199 problem.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -59,7 +59,7 @@ services:
     volumes:
       - .:/app
       - node_modules:/app/node_modules
-    command: webpack-dev-server --hot --inline --progress --host 0.0.0.0 --config config/webpack.config.js
+    command: yarn webpack
```
Comment (talyssonoc) [inline docker-compose.yml]: Shouldn't this be `yarn run webpack` here?

Comment (leonardofreitass) [inline docker-compose.yml]: Actually no, just `yarn webpack` works fine. Do you think that `yarn run webpack` is better?

Comment (talyssonoc) [inline docker-compose.yml]: I'd use the pattern of using `yarn <command>` (without the `run`) only for the default used by npm too (`start`, `test` and others...)

Comment (leonardofreitass) [inline docker-compose.yml]: Done!

**Pull request merged**